_data/publications/1738733.yml

```
@@ -0,0 +1,2 @@
+inspire-id: 1738733
+project: MkFit
```

```suggestion
project: mkfit
```

It's safer to match the project short name exactly. I might be able to always lower the case, but first I'd want a check to make sure all project short names are lower case.